python/lsst/ts/rubintv/handlers/handlers_helpers.py

```diff
@@ -80,6 +80,8 @@ async def get_camera_events_for_date(
     metadata = await historical.get_metadata_for_date(location, camera, day_obs)
     per_day = await historical.get_per_day_for_date(location, camera, day_obs)
     nr_exists = await historical.night_report_exists_for(location, camera, day_obs)
+    if (channel_data == metadata == per_day == {}) and nr_exists is False:
```

Comment:
I recommend to use logical notations by using De Morgan's Law:
```suggestion
    if not (channel_data or metadata or per_day or nr_exists):
```

Reply:
Thanks Sebastian! That's a lot more elegant 👍